test/src/tests-core-sdk.js

```diff
@@ -924,6 +923,7 @@ describe('core SDK', function() {
         'minWebviewBridgeVersion',
         'aliasMaxWindow'
     ];
+    // TODO: Refactor this test to respect type safety or remove as this duplicates tests in store.ts
```

Comment:
I'm fine with that.  I do remember writing a test and I suppose I put it in here instead of a `test-store.js` because I figured it was core to the SDK, and it was a single test.  since this is more an integration test I think we should move it over and label it as such.